### DISCUSSION TITLE
Configuring cloudflare

### DISCUSSION THREAD

**Discussion by cmtewerneck** [vercel/community#5752] [Help]
### Summary

Hi everybody, i really need a help. I am a begginer dev and i have created a cloudflare account and pinted my domain to cloudflare servers. It is ok! But the problem is. I have created a CNAME record pointing cloudflare to vercel but my website is not working, somethins is wrong. (see my registers on image)
![asdas](https://github.com/vercel/community/assets/63486472/00dc13d9-89c2-4060-98f4-c7886baf2076)

### Steps to Reproduce

I would like to know how to make this configuration!

**Comment by amyegan** [comment]
Hey @cmtewerneck! We have a guide available to guide you through the setup steps for this: [Vercel and Cloudflare Integration](https://vercel.com/docs/integrations/cloudflare#)

Please try that and let me know if you still have trouble 🙏

**Reply by cmtewerneck** [reply]
Hey @amyegan thanks a lot, but can you help me? I have removed all "A" register and inserter the CNAME register as you can see on the image i have sent on the post. I did this following this article but it is not working. I dont know if i need to add something more.
